src/app/switcher/switcher-program/switcher-program.component.html

```diff
@@ -1,20 +1,20 @@
 <ion-header mode="ios">
   <ion-toolbar>
-    <ion-title aria-label="select an experience" role="title" tabindex="0">Select an experience</ion-title>
+    <ion-title aria-label="select an experience"  tabindex="0" id="test-title">Select an experience</ion-title>
```

Comment:
Can we use something like 'page-title' instead of 'test-title'.

Reply:
thanks @sasangachathumal , sure, sorry didn't check that.